src/realm/query_conditions.hpp

```diff
@@ -112,6 +113,17 @@ struct Contains : public HackClass {
         return v2.contains(v1, charmap);
     }
 
+    bool operator()(const Mixed& m1, const Mixed& m2, bool = false, bool = false) const
```

Comment:
It looks like this code is copy/pasted 10 times?

Reply:
Yes, it is. Not sure it could be done differently.

Reply:
I have no clue about the context this is in... Why can't this be in a function?

Reply:
It is a function in a class template and the function uses other functions defined in that template.